scheds/include/scx/common.bpf.h

```diff
@@ -35,6 +35,10 @@
 #define PF_EXITING			0x00000004
 #define CLOCK_MONOTONIC			1
 
+#ifndef NR_CPUS
+#define NR_CPUS 512
```

Comment:
Current gen epyc already have almost [400 cpus](https://www.amd.com/en/products/processors/server/epyc/9005-series/amd-epyc-9965.html), maybe bump this to 1024?